types/tress/tress-tests.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="node" />
 
-import { tress } from "tress";
```

Comment:
How was this ever working for anyone?

Reply:
@DanielRosenwasser, it wasn't. But I only paid attention to the odds of my first version of tress' definition at the moment I started using it in a Typescript project.